apps/users/views.py

```diff
@@ -22,23 +22,22 @@ def register(request):
     """
     if request.method == "POST":
         user_form = MemberCreateForm(request.POST)
+        profile_form = ProfileCreateForm(request.POST)
```

Comment:
can you add a test where it would fail before this merge but would pass after the merge?

Reply:
done